portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -141,9 +141,9 @@
 	<td></td>
 </tr>
 <tr>
-   <td>FIELD_VALIDATION_VALUE</td>
-   <td>//div[contains(@data-field-name,'validation')]//label[contains(text(),'${key_fieldLabel}')]</td>
-   <td></td>
+<td>FIELD_VALIDATION_VALUE</td>
+<td>//div[contains(@data-field-name,'validation')]//label[contains(text(),'${key_fieldLabel}')]</td>
+<td></td>
```

Comment:
@gleicelisbino, it shouldn't be removed.

Reply:
Okay! @natocesarrego